src/gui/torrentcontentmodel.cpp

```diff
@@ -39,299 +39,297 @@
-  if (parentItem == m_rootItem)
-    return QModelIndex();
+    TorrentContentModelItem *parentItem = childItem->parent();
+    if (parentItem == m_rootItem)
+        return QModelIndex();
 
-  return createIndex(parentItem->row(), 0, parentItem);
+    return createIndex(parentItem->row(), 0, parentItem);
 }
 
 int TorrentContentModel::rowCount(const QModelIndex& parent) const
 {
-  if (parent.column() > 0)
-    return 0;
+    if (parent.column() > 0)
+        return 0;
 
-  TorrentContentModelFolder* parentItem;
-  if (!parent.isValid())
-    parentItem = m_rootItem;
-  else
-    parentItem = dynamic_cast<TorrentContentModelFolder*>(static_cast<TorrentContentModelItem*>(parent.internalPointer()));
+    TorrentContentModelFolder* parentItem;
+    if (!parent.isValid())
+        parentItem = m_rootItem;
+    else
+        parentItem = dynamic_cast<TorrentContentModelFolder*>(static_cast<TorrentContentModelItem*>(parent.internalPointer()));
 
-  return parentItem ? parentItem->childCount() : 0;
+    return parentItem ? parentItem->childCount() : 0;
 }
 
 void TorrentContentModel::clear()
 {
-  qDebug("clear called");
-  beginResetModel();
-  m_filesIndex.clear();
-  m_rootItem->deleteAllChildren();
-  endResetModel();
+    qDebug("clear called");
+    beginResetModel();
+    m_filesIndex.clear();
+    m_rootItem->deleteAllChildren();
+    endResetModel();
 }
 
 void TorrentContentModel::setupModelData(const BitTorrent::TorrentInfo &info)
 {
-  qDebug("setup model data called");
-  if (info.filesCount() == 0)
-    return;
-
-  emit layoutAboutToBeChanged();
-  // Initialize files_index array
-  qDebug("Torrent contains %d files", info.filesCount());
-  m_filesIndex.reserve(info.filesCount());
-
-  TorrentContentModelFolder* current_parent;
-  // Iterate over files
-  for (int i = 0; i < info.filesCount(); ++i) {
-    current_parent = m_rootItem;
-    QString path = Utils::Fs::fromNativePath(info.filePath(i));
-    // Iterate of parts of the path to create necessary folders
-    QStringList pathFolders = path.split("/", QString::SkipEmptyParts);
-    pathFolders.removeLast();
-    foreach (const QString& pathPart, pathFolders) {
-      if (pathPart == ".unwanted")
-        continue;
-      TorrentContentModelFolder* new_parent = current_parent->childFolderWithName(pathPart);
-      if (!new_parent) {
-        new_parent = new TorrentContentModelFolder(pathPart, current_parent);
-        current_parent->appendChild(new_parent);
-      }
-      current_parent = new_parent;
+    qDebug("setup model data called");
+    if (info.filesCount() == 0)
+        return;
+
+    emit layoutAboutToBeChanged();
+    // Initialize files_index array
+    qDebug("Torrent contains %d files", info.filesCount());
+    m_filesIndex.reserve(info.filesCount());
+
+    TorrentContentModelFolder* current_parent;
+    // Iterate over files
+    for (int i = 0; i < info.filesCount(); ++i) {
+        current_parent = m_rootItem;
```

Comment:
@sledgehammer999 believes that we should relax this requirement, but I disagree. At least, we should use camelCase in all new code and in "Follow project coding style. Issue #2192" commits.
It is not so difficult to fix. You just need to search for ```_``` and do **Refactor => Convert to Camel Case**. No more than 5-10 minutes even for large files.